README.md

```diff
@@ -4,6 +4,16 @@ A repository for developing the User Interface for receiving data and controllin
 
 ## Packages Info
 
+### Whole Launch
+
+1.	Open your terminal and navigate to your working directory (for example, ros2_ws).
+2.	Clone the latest version of the MRCS: `git@github.com:GonzagaRobotics/ManualRoverControlSystem.git`
+3.	Once the repository is cloned, navigate to the top of the ros2_ws directory and build the project using Colcon by running the command: `colcon build`. If an error occurs, try again until it works.
```

Comment:
Is there any specific actions the user should take if an error occurs? If they've followed the previous steps exactly, would an error occur? If the answer is "it depends", try to ask what it depends on, and if you can/if it is reasonable to instruct the user on the specific setup necessary to avoid an error.